src/useMintUsd.ts

```diff
@@ -26,6 +25,7 @@ export function useMintUsd({
 }) {
   const { chainId, queryClient } = useSynthetix();
   const { data: systemToken } = useImportSystemToken();
+  const { data: priceIds } = useAllPriceFeeds();
```

Comment:
Same here.

Reply:
Removed  `useAllPriceFeeds()`.
Added `usePriceUpdateTxn()`:

`const { data: priceUpdateTxn } = usePriceUpdateTxn({ provider });`

Replaced cache cleaning: 
from:  `queryKey: [chainId, 'PriceUpdateTxn', { priceIds: priceIds?.map((p) => p.slice(0, 8)) }]`
to:  `queryKey: [chainId, 'PriceUpdateTxn']`